bitnami/mongodb-sharded/values.yaml

```diff
@@ -18,7 +18,7 @@ global:
   ##   - myRegistryKeySecretName
   ##
   imagePullSecrets: []
-  storageClass: ""
+  defaultStorageClass: ""
```

Comment:
You didn't update the value metadata in this case

Reply:
fixed